straxen/plugins/peaklets_events/peaklets_events.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+import straxen
+import strax
+
+export, __all__ = strax.exporter()
+
+
+@export
+class PeakletsEvents(strax.Plugin):
+    """
+    Concatenate peaklets that constitute the main and alternative S1s and S2s.
+    Allow waveform watching by analysts, while significantly decrease storage pressure.
+    """
+
+    depends_on = ('peaklets', 'event_basics')
+    provides = ('peaklets_events')
+    data_kind = ('peaklets_events')
+    parallel = 'process'
+    compressor = 'zstd'
+    save_when = strax.SaveWhen.ALWAYS
+
+    __version__ = '0.0.1'
+
+    n_tpc_pmts = straxen.URLConfig(
+        type=int,
+        help='Number of TPC PMTs')
+
+    sum_waveform_top_array = straxen.URLConfig(
+        default=True,
+        type=bool,
+        help='Digitize the sum waveform of the top array separately'
+    )
+
+    def setup(self):
+        return
+
+    def infer_dtype(self):
+        return strax.peak_dtype(
+            n_channels=self.n_tpc_pmts,
+            digitize_top=self.sum_waveform_top_array,
+        )
+
+    def fake_mask(self, df, start_field='time', end_field='end_time', pad=10e3):
+        """
+        Create a fake mask to touch the peaklets
+        :param df: events
```

Comment:
I would call it `events` and not `df` for the sake of clean code and readability. If you call it events I know directly what the object is suppose to represent. Like it is know I will think that I have to provide a dataframe here which is most likely wrong as plugins handle data in from of structure arrays. 

In addition I think the entire function is not needed. See my comment below.